frontend/src/components/CharacterCreation/CharacterInfo.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactElement } from 'react';
+import * as CLASSES from './classes';
+// Need to look into how i'm importing.  I think doing it this way is making it harder.
+import { SKILLS } from './skills';
+
+interface Props {
+  currentClass: string;
```

Comment:
this should be `currentClass: 'Knight' | 'Rogue' | 'Barbarian' | 'Mage'` which can be an exported type from your classes file

Reply:
Updated so that most var have a type